src/DataFrames.jl

```diff
@@ -80,6 +80,13 @@ if VERSION < v"1.2"
     export hasproperty
 end
 
+if VERSION >= v"1.4"
```

Comment:
I'd probably do this as
```suggestion
if isdefined(Base, :only)  # Introduced in 1.4.0
```
Probably matters much less nowadays than it used to in the Julia 0.x days. You're welcome to ignore this suggestion if you prefer what's currently there.

Reply:
Actually I was considering this myself, but then I saw what we had for other methods. I am OK to change to what you propose.